src/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!--
+        This file would be included in all projects automatically.
+        AssemblyCommon.props included only in SDK projects, because non-SDK projects does not support PackageReferences and some othrer properties.
```

Comment:
nit:
```suggestion
        This file is included in all projects automatically.
        AssemblyCommon.props is included only in SDK projects, because non-SDK projects do not support PackageReferences and some other properties.
```